locales-he-IL.xml

```diff
@@ -23,29 +23,29 @@
     <term name="and">ו</term>
     <term name="and others">ואחרים</term>
     <term name="anonymous">אלמוני</term>
-    <term name="anonymous" form="short">anon</term>
+    <term name="anonymous" form="short"></term>
     <term name="at">-ב</term>
     <term name="available at">זמין ב</term>
     <term name="by">על-ידי</term>
     <term name="circa">לערך</term>
-    <term name="circa" form="short">c.</term>
+    <term name="circa" form="short">~</term>
     <term name="cited">מצוטט ב</term>
     <term name="edition">
       <single>מהדורה</single>
       <multiple>מהדורות</multiple>
     </term>
-    <term name="edition" form="short">ed</term>
+    <term name="edition" form="short">מהדורה</term>
     <term name="et-al">ואחרים</term>
     <term name="forthcoming">צפוי</term>
     <term name="from">מתוך</term>
     <term name="ibid">שם</term>
     <term name="in">בתוך</term>
-    <term name="in press">בהדפסה</term>
-    <term name="internet">אינטרנט</term>
+    <term name="in press">בדפוס</term>
+    <term name="internet">אתר</term>
     <term name="interview">ראיון</term>
     <term name="letter">מכתב</term>
-    <term name="no date">אין נתונים</term>
-    <term name="no date" form="short">nd</term>
+    <term name="no date">לא ידוע</term>
+    <term name="no date" form="short"></term>
```

Comment:
Needs a translation.

Reply:
as per above

Reply:
if the no date is referring specifically to publishing details, then:
חש"ד, which is an acronym for חסר שנת דפוס (lit. missing the year of printing). In Hebrew, however, only the acronym is ever used.